tests/managedmcg_controller_test.go

```diff
@@ -6,13 +6,15 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	managedmcgv1alpha1 "github.com/red-hat-storage/mcg-osd-deployer/api/v1alpha1"
+	"github.com/red-hat-storage/mcg-osd-deployer/controllers"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-)
 
-const (
-	Namespace = "openshift-storage"
```

Comment:
Let's keep the `namespace` here itself. `suite_test.go` is meant for setting up extraneous setup logic for the fake cluster, and it's better to couple similar logic under the same test file.

Reply:
Okay.

Reply:
@rexagod , Here is an issue while creating and deleting separate namespace for both specs. As both specs are executing in parallel and we are trying to accessing the ns from one spec which was already in terminating state by other spec.

Reply:
Parallelization only happens if we pass the `-p` flag to Ginkgo, which is not the case right now. Are you sure about this?

Reply:
Tests should themselves be mutually exclusive in execution, and any race conditions should be avoided while writing them. A good practice is to imagine the the specs will execute in parallel, even if that's not actually the case, and write your spec based on that logic. In this case NS creation and deletion would happen for both specs, if they require it to exist.